### PR TITLE
Adding platform.name = "extension" to be used by the new navigator interface

### PR DESCRIPTION
**Pull request by shakyShane**
**Reviewer:** @jonathanKingston 

Note: This PR changed in scope since the bulk of the logic will now be implemented in https://github.com/duckduckgo/content-scope-scripts

This PR is now just here to provide `platform.name` to the args that get passed to the content scripts. 

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [x] Get advice or leverage existing code
- [x] Agree on technical approach with reviewer (if the changes are nuanced)
- [x] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: integration-test/content-scripts/test-canvas.js [hunk 1]
```diff
@@ -115,8 +98,8 @@ let server2
 describe('First Party Fingerprint Randomization', () => {
     beforeAll(async () => {
         ({ browser, bgPage } = await harness.setup())
-        server = setupServer({}, 8080)
```
Comment (jonathanKingston) [inline integration-test/content-scripts/test-canvas.js:118]: Most of this test noise goes away with a rebase right?

**Comment by jonathanKingston** [comment]
So clean!

**Comment by shakyShane** [comment]
@jonathanKingston this is tiny now, but I'm curious if you think this warrants any automated tests on it's own?

**Comment by jonathanKingston** [comment]
@shakyShane yeah I think you had some right? Can they be moved into the shared repo?

**Comment by shakyShane** [comment]
> 

yep :)

**Review by jonathanKingston — APPROVED** [review]

**Pull request merged**